app/teams/page.tsx

```
@@ -0,0 +1,104 @@
+                            <div className="font-mono text-lg">
+                              {prettyMs(Number(avg_proving_time) || 0)}
+                            </div>
+                          </div>
+                          <div className="flex flex-col items-center gap-2 px-4">
+                            <div className="flex items-center gap-1 text-body-secondary">
+                              {AVERAGE_LABEL} <metrics.costPerProof.Label />
+                            </div>
+                            <div className="font-mono text-lg">
+                              {avg_cost_per_proof !== null &&
+                              avg_cost_per_proof !== 0 &&
+                              isFinite(avg_cost_per_proof) ? (
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)